servicetalk-client-api/src/main/java/io/servicetalk/client/api/ServiceDiscoverer.java

```diff
@@ -39,8 +44,12 @@ public interface ServiceDiscoverer<UnresolvedAddress, ResolvedAddress,
      * {@code address}.
      * <p>
      * In general a call to this method will continue to discover changes related to {@code address} until the
-     * {@link Subscription} corresponding to the return value is cancelled via {@link Subscription#cancel()} or there
-     * are no more changes to be published.
+     * {@link Subscription Subscription} corresponding to the return value is {@link Subscription#cancel() cancelled} or
```

Comment:
```suggestion
     * {@link Subscription} corresponding to the return value is {@link Subscription#cancel() cancelled} or
```

Reply:
This is intentional bcz without an alias javadoc puts a full class name, which breaks readability of the javadoc

Reply:
Ah, I see.